### PR TITLE
changing timestamp to stageTimestamp as kubernetes audit log format has changed

### PR DESCRIPTION
**Pull request by aibitayo55555**
…as changed

### CODE REVIEW

File: conf.d/20_output_kubernetes_audit_elasticsearch.conf [hunk 1]
```diff
@@ -30,7 +30,7 @@ filter {
 
     # Extract timestamp of the acutal event and overwrite @timestamp
     date {
-      match => [ "[audit_json][timestamp]", "ISO8601" ]
+      match => [ "[audit_json][stageTimestamp]", "ISO8601" ]
```
Comment (marcinc) [inline conf.d/20_output_kubernetes_audit_elasticsearch.conf:33]: Should rather be `requestReceivedTimestamp` as we mostly interested of when request hit the api.

**Review by marcinc — CHANGES_REQUESTED** [review]

File: conf.d/20_output_kubernetes_audit_elasticsearch.conf (hunk 1 above)
Comment (marcinc) [reply conf.d/20_output_kubernetes_audit_elasticsearch.conf:33]: Actually you were right in the original PR. There can be multiple events in the log for the same API call and it's attributed to stage change. So please revert back to what you did initially. Thanks.

**Review by marcinc — CHANGES_REQUESTED** [review]
Actually you were right in the original PR. There can be multiple events in the log for the same API call and it's attributed to stage change. So please revert back to what you did initially. Thanks.

File: conf.d/20_output_kubernetes_audit_elasticsearch.conf (hunk 1 above)
Comment (aibitayo55555) [reply conf.d/20_output_kubernetes_audit_elasticsearch.conf:33]: Thanks I have changed it back

Comment (marcinc) [reply conf.d/20_output_kubernetes_audit_elasticsearch.conf:33]: 👍

**Review by marcinc — APPROVED** [review]
LGTM

**Pull request merged**